composer.json

```
@@ -33,7 +33,7 @@
         "symfony/console": "^5.3.0",
         "symfony/dotenv": "^5.3.0",
         "symfony/expression-language": "^5.3.0",
-        "symfony/flex": "^1.11",
+        "symfony/flex": "^1.11|^2",
```

See: https://getcomposer.org/doc/articles/versions.md#version-range
```suggestion
        "symfony/flex": "^1.11 || ^2",
```

---

Thanks, changed in https://github.com/ibexa/oss/pull/44/commits/ca86227aec9b1f1fb5a15aa1e0620346b379b367